flake.nix

```diff
@@ -7,17 +9,30 @@
 
     flake-parts.url = "github:hercules-ci/flake-parts";
     nixos-flake.url = "github:srid/nixos-flake";
+    systems.url = "github:nix-systems/default";
   };
 
-  outputs = inputs@{ self, ... }:
+  outputs =
+    inputs @ { self
+    , nixpkgs
+    , home-manager
+    , flake-parts
+    , nixos-flake
+    , systems
+    , ...
+    }:
```

Comment:
Let's go with `outputs = inputs: ...` and then reference the inputs in qualified manner (eg: `inputs.systems`).

Reply:
https://github.com/cameronraysmith/nix-dev-home/commit/f994613302818ec296e5b5e19fba44342c0d1d3d